te_dp/src/te_stats_collector.cpp

```diff
@@ -997,8 +997,12 @@ int main(int argc, char** argv)
         exit(1);
     }
 
-    freopen("/tmp/te_stats_collector_output.log","w",stdout);
-    freopen("/tmp/te_stats_collector_error.log","w",stderr);
+    FILE* fp;
```

Comment:
Make was complaining about not handling the return of `freopen`. Fixing the warning.